content/systems/filecoin_blockchain/chainsync/_index.md

```diff
@@ -1,42 +1,52 @@
 ---
 title: ChainSync
 weight: 3
-description: ChainSync - synchronizing the Blockchain
 dashboardWeight: 2
 dashboardState: reliable
 dashboardAudit: missing
 dashboardTests: 0
 ---
 
-# ChainSync - synchronizing the Blockchain
-
-## What is blockchain synchronization?
+# ChainSync
 
 Blockchain synchronization ("sync") is a key part of a blockchain system.
 It handles retrieval and propagation of blocks and transactions (messages), and
 thus is in charge of distributed state replication.
-**This process is security critical -- problems here can be catastrophic to the
-operation of a blockchain.**
+As such, this process is security critical -- problems with state replication can have severe consequences to the
+operation of a blockchain.
 
-## What is ChainSync?
+## ChainSync Overview
 
 `ChainSync` is the protocol Filecoin uses to synchronize its blockchain. It is
 specific to Filecoin's choices in state representation and consensus rules,
 but is general enough that it can serve other blockchains. `ChainSync` is a
 group of smaller protocols, which handle different parts of the sync process.
 
+Chain synchronisation generally applies to three main processes:
+1. Chain synchronization when a node first joins the network and needs to get to the current state before validating or extending the chain.
+2. Chain synchronisation when a node has fell out of sync, e.g., due to a brief disconnection.
+3. Continuous chain synchronisation to keep up with the latest messages and blocks.
```

Comment:
These 3 are the same process. The differentiation is useful as examples of how the sync behaves in different circumstances but let's not call each of them processes.